### PR TITLE
Added description and parameters

### PR DESCRIPTION
**Pull request by alorne**

### CODE REVIEW

File: data/en/gettimezone.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
 	"syntax": "getTimezone()",
 	"returns": "string",
 	"related": ["getTimezoneInfo","setTimezone","getLocale"],
-	"description": "",
-	"params": [],
+	"description": "Returns the timezone for the current request.",
+	"params": ["This function does not take any arguments"],
```
Comment (shaedrich) [inline data/en/gettimezone.json]: Please don't do that. This will break the doc.

Comment (alorne) [reply data/en/gettimezone.json]: I'm sorry, I'm new at this. What exactly is wrong?

Comment (shaedrich) [reply data/en/gettimezone.json]: No worries! `params` is either an empty array or a array of structs. So if you'd pass a string it considers it an non-empty array, tries to loop over the structs and wonders why it gets a simple value.

Comment (alorne) [reply data/en/gettimezone.json]: Ok, I'll fix it. Thanks!

Comment (shaedrich) [reply data/en/gettimezone.json]: Thanks! You're welcome!

**Comment by pfreitag** [comment]
Thanks!

**Pull request merged**